data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -108,7 +109,7 @@ public Results executeStreamingIngest(String database, String table, InputStream
         if (StringUtils.isBlank(streamFormat)) {
             throw new IllegalArgumentException("Parameter streamFormat is empty.");
         }
-        String clusterEndpoint = String.format("%s/%s/rest/ingest/%s/%s?streamFormat=%s", clusterUrl, API_VERSION, database, table, streamFormat);
+        String clusterEndpoint = String.format("%s/%s/rest/ingest/%s/%s?streamFormat=%s", clusterUrl, QUERY_ENDPOINT_VERSION, database, table, streamFormat);
```

Comment:
Streaming ingest version is v1.
I would have added a member in the StreamingClient to hold the streaming ingest version.

Reply:
cant hold it in StreamingClient as its in the ingest package